src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
The enum values should be declared in upper case as it is constants. Please, refer to [this](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) guide.